README.md

```diff
@@ -4,6 +4,8 @@
 
 A set of scripts to configure and run a dockerized algod for [fnet](https://fnet.algorand.green/).
 
+**Entry point: `./run.sh`**. Run this to configure and start your container.
```

Comment:
Maybe a traditional Get Started section. Entry point could be confused with the Docker Entrypoint

# Get Started

Use the `run` script to configure and launch a node

```bash
./run.sh
```

Reply:
done